Formula/wazero.rb

```diff
@@ -0,0 +1,67 @@
+class Wazero < Formula
+  desc "Zero dependency WebAssembly runtime"
+  homepage "https://wazero.io"
+  url "https://github.com/tetratelabs/wazero/archive/v1.0.1.tar.gz"
+  sha256 "9cad65fec1043060e77d303765ba7448ba1dce5a4149e830c02cc56d0f72f00f"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "wabt" => :test
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/tetratelabs/wazero/internal/version.version=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/wazero/..."
+  end
+
+  test do
```

Comment:
Since we're specifying the version via a flag in `install`, let's also check that the output of `--version` (or equivalent) is what we expect given the flag we used.

Reply:
can do!